src/main/java/fr/iut/pathpilotapi/client/ClientService.java

```diff
@@ -18,6 +20,8 @@ public class ClientService {
 
     private final ClientCategoryRepository clientCategoryRepository;
 
+    private final SalesmanService salesmanService;
+
     /**
      * Get all clients from the database
      *
```

Comment:
`getAllClient` should only return the clients of the Salesman who request it.

Reply:
done